justfile

```diff
@@ -45,20 +45,18 @@ ENABLE_DC_OVERRIDES := env_var_or_default("OPENVERSE_ENABLE_DC_OVERRIDES", "true
 #######
 
 # Install Node.js dependencies for the entire monorepo
-node-install:
+node-install: env
     pnpm i
     just frontend/run i18n:en
     just frontend/run i18n:copy-test-locales
 
 # Install Python dependences for the monorepo
-py-install:
+py-install: env
     just automations/python/install
     just documentation/install
 
 # Install all dependencies
-install:
-    just node-install
-    just py-install
+install: node-install py-install
```

Comment:
This is not aligned with the general convention in our `justfile`s. Alternatively we could adopt this and then there's a lot more instances of recipes just being a collection of other `just` recipe calls.